### DISCUSSION TITLE
Audio duration sometimes not showing

### DISCUSSION THREAD

**Discussion by totti-rdz** [lhz516/react-h5-audio-player#172] [Q&A]
Hi, first thanks to the creator for the great work! 🙏

I'm using the audio player in one of my projects with Next.js and Tailwind CSS and I've already noticed that sometimes the duration for a track is not shown. Only the placeholder `--:--` at the end of the progress bar. At first I thought that this is maybe a network related issue, e.g. the duration can not be shown until the track is fully downloaded, but I'm not sure about that as sometimes I have the player open for a while.

I created a show project only with the audioplayer and one media file from wikipedia for another bug I wanted to inspect which turned out to be a feature in iOS (volume control not working) and the missing duration behaviour is showing here again. But sometimes it does.

The duration is missing on Mozilla Firefox 104.0.2 (64-bit) macOS. On Chrome (macOS), Brave browser (android) and Safari (iOS) the duration is shown.

Please visit:
https://react-h5-audio-player.vercel.app/

https://github.com/totti-rdz/react-h5-audio-player

**Comment by lhz516** [comment]
Yea, it happens quite randomly. Could you create an issue so that we can track it better?

**Reply by totti-rdz** [reply]
done
👇
https://github.com/lhz516/react-h5-audio-player/issues/177

**Reply by luantran24hdev** [reply]
it's not work. Please help me
